packages/storybook-addon-baseline-grid/src/manager.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { addons, types } from "storybook/internal/manager-api";
+
+import { Tool } from "./components/Tool";
+import { ADDON_ID, TOOL_ID } from "./constants";
+
+/**
+ * Note: if you want to use JSX in this file, rename it to `manager.tsx`
+ * and update the entry prop in tsup.config.ts to use "src/manager.tsx",
+ */
+
+// Register the addon
+addons.register(ADDON_ID, (api) => {
+  // Register a tool
+  addons.add(TOOL_ID, {
+    type: types.TOOL,
+    title: "Baseline grid addon",
+    match: ({ viewMode }) => !!(viewMode && viewMode.match(/^(story)$/)),
```

Comment:
As-is, this shows the baseline button when viewed in a storybook window, but not when the story is opened full-screen, which means we can't easily toggle the baseline in full-screen mode.

![Screenshot 2024-12-19 at 16 21 40](https://github.com/user-attachments/assets/7c06d8b5-ee50-4858-a309-24b816c4855a)

If it's possible to trivially fix that, I'd suggest adding it, otherwise we can leave it for later

Reply:
This part was generated and based on an example Tool addon they provided.

I created a follow up to define where we want the addon to be available #93

(I'm not sure if any toolbar addons are avaialble in fullscreen mode?)

Reply:
I'll add this to the discussion in #95